openvino/cc/scripts/ccheader.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+
+import argparse, csv
```

Comment:
I think it is better hide script in cmake custom command.

Reply:
If we hide this script inside CMake user can't use it with custom application.

Reply:
Why user should use it for custom application? First of all we solve the problem of OpenVINO conditional compilation.

We already have discussed it, and:

-  looks like it will be easier to use the CMake command (it allow to avoid additional steps).
- We always can make this script public and update the documentation (it is easier than hide the script in the future)
- Advanced users can easily find and use this script, but we don't have additional problems with backward compatibility if it is our internal solution.

Reply:
1. looks like it will be easier to use the CMake command (it allow to avoid additional steps).
We hide only one step.
2. Ok
3. Advanced users can easily find and use this script...
Unfortunately, he can't use the script because his generated definitions in the header file will conflict with our header, which will be hidden inside Cmake.

Reply:
Ok, I'll hide header file generation step inside CMake.

Reply:
done